robottelo/ui/base.py

```diff
@@ -130,6 +130,31 @@ def search(self, element_name):
             time.sleep(1)
         return None
 
+    def create_a_bookmark(self, name=None, query=None, public=None,
```

Comment:
Bookmarks don't have their own creation page. Every entity (with really rare exclusions) has a searchbox with ability to make a bookmark. And the bookmark creation flow is the same for each entity (except the difference for katello/non-katello, which was eliminated with some tricky xpaths). So it's pretty obvious that there should be some `create_a_bookmark` method in parent class, which every entity should inherit, adding such method to each entity is a bad idea. I know _Base_ class isn't the best place for it, Base class should contain only the low-level things, we already discussed that before. But that's our design flaw which i hope we'll address in future. Right now our `Base` class contains everything what's useful for every child entity, so that's the best place to put `create_a_bookmark()` ATM.